Automattic/wp-calypso
# Plans step redesign: Implement the CTA button

#### Changes proposed in this Pull Request

This PR implements the experiment that redesigns the signup flow plans step. Users in the treatment group will see the new action buttons for each plan in the signup plans step.

NOTE: 
1. This PR uses the changes in https://github.com/Automattic/wp-calypso/pull/49904 as the base branch and is built on top of it.
2. This PR only implements the CTA buttons section. The rest of the plan sections are implemented in subsequent PRs.

**Important links**
1. [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1362%3A813)
2. pau2Xa-2yE-p2 for the project thread.

<img width="1654" alt="Screenshot 2021-02-15 at 7 34 47 PM" src="https://user-images.githubusercontent.com/1269602/107956243-e28b8680-6fc4-11eb-8c71-b137d6d7768e.png">


#### Testing instructions



* Open http://calypso.localhost:3000/ and assign yourself to treatment of the `signup_plans_step_redesign_v1` experiment in the Abacus tool.
* Begin signup at `/start` and verify that the plans step UI matches the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1362%3A813). Verify for both free subdomain and custom domain selections.
* Click on any of the CTA buttons and confirm that you proceed to checkout with the plan added to cart.



Related to https://github.com/Automattic/wp-calypso/pull/49877
